backend/sessao/api/serializers.py

```diff
@@ -30,17 +28,24 @@ class Meta:
             "totalDeCamisetas",
             "velocidade",
             "base_producao_create",
-            "base_producao_get"
+            "base_producao_get",
+            "image"
         )
 
     def create(self, validated_data):
         producao = models.Producao.objects.create_from_json(validated_data)
         return producao
 
-
 class LoteSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Lote
         fields = '__all__'           
 
+    def update(self, instance, validated_data):
+        instance.image = validated_data.get('image', instance.image)
+        ##implementar aqui a validação da camiseta
```

Comment:
Está faltando essa parte mencionada no comentário ?

Reply:
Essa parte do código foi feita no início. No decorrer do desenvolvimento, utilizamos outra estratégia. Já removi esse trecho de código.